app/assets/javascripts/Components/Result/result.jsx

```diff
@@ -29,6 +30,13 @@ class Result extends React.Component {
       feedback_files: [],
       submission_files: {files: [], directories: {}, name: '', path: []},
       fullscreen,
+      annotationModal: {
```

Comment:
This "initial annotationModal state" is used a couple of places in this file. SInce all the values are static, would be good to pull this out into a constant.